modules/oadp-incremental-backup-support.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+// backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc
+// backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-about-incremental-backup-support_{context}"]
+= About incremental back up support
+
+{oadp-short} supports incremental backups of `block` and `Filesystem` persistent volumes for both containerized, and {VirtProductName} workloads. The following table summarizes the support for File System Backup (FSB), Container Storage Interface (CSI), and CSI Data Mover:
+
+[cols="5", options="header"]
+.{oadp-short} backup support matrix for containerized workloads
+|===
+| Volume mode |FSB - Restic  |FSB - Kopia | CSI | CSI Data Mover
+| Filesystem | S ^[1]^, I ^[2]^ | S ^[1]^, I ^[2]^ | S ^[1]^ | S ^[1]^, I ^[2]^
+| Block | N ^[3]^ | N ^[3]^ | S ^[1]^ | S ^[1]^, I ^[2]^
+|===
+
+[cols="5", options="header"]
+.{oadp-short} backup support matrix for {VirtProductName} workloads
+|===
+| Volume mode |FSB - Restic  |FSB - Kopia | CSI | CSI Data Mover
+| Filesystem | N ^[3]^ | N ^[3]^ | S ^[1]^ | S ^[1]^, I ^[2]^
+| Block | N ^[3]^ | N ^[3]^ | S ^[1]^ | S ^[1]^, I ^[2]^
+|===
+
+[.small]
+--
+1. Backup supported
+2. Incremental backup supported
+3. Not supported
+--
```

Comment:
Refer to the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables) for footnotes and table formatting.

Reply:
The table is majorly compliant apart from the [.small] style. I have seen the [.small] style at a couple of places in OADP and it works fine so far. If I remove the [.small] style, the footnote descriptions appear in a font bigger than the table contents. Thanks.